cluster_management/src/main/java/com/pinterest/rocksplicator/Participant.java

```diff
@@ -170,6 +170,9 @@ public Participant(String zkConnectString, String clusterName, String instanceNa
     } else if (stateModelType.equals("MasterSlave")) {
       stateModelFactory = new MasterSlaveStateModelFactory(instanceName.split("_")[0],
           port, zkConnectString, clusterName);
+    } else if stateModelType.equals("BulkIngestion") {
```

Comment:
use a better name than BulkIngestion? which doesn't indicate data could be online updated.

Reply:
I think we should probably meet in person to discuss this diff, it is actually intended for bulkingestion and not for online updates